yo1k/tic_tac_toe/test/kernel/test_game.py

```diff
@@ -54,16 +58,16 @@ class LogicSingleActionTest(unittest.TestCase):
     def test_advance__no_action(self) -> None:
         state = _new_state(board=Board([
                 [None, None, None], [None, None, Mark.X], [None, None, None]]))
-        Logic((ListActionQueue([]), ListActionQueue([]))).advance(state)
+        Logic((ListActionQueue(PlayerID(0), []), ListActionQueue(PlayerID(1), []))).advance(state)
```

Comment:
I realized that we can use `state.players[0].id`/`state.players[1].id` everywhere in tests instead of creating new `PlayerID` objects.

Reply:
Done.